test/acceptance/features/bindAppToService.feature

```diff
@@ -273,4 +273,37 @@ Feature: Bind an application to a service
             status:
                 ready: true
             """
-        Then Secret "binding-request-backend" contains "CustomReady" key with value "true"
\ No newline at end of file
+        Then Secret "binding-request-backend" contains "CustomReady" key with value "true"
+
+    Scenario: Ensure custom environment variable inject without any prefix
```

Comment:
How about rephrasing to:

```
Custom environment variable is injected into the application under the declared name ignoring global and service env prefix
```

It reads better IMHO.

Reply:
Done.